src/simd_nf4.h

```diff
@@ -52,19 +54,18 @@ static inline aint128 m128i_to_uint128(m128i v)
 inline aint128 expand16(uint16_t* arr, int n)
 {
     // since n <= 4
-    uint16_t _arr[4] __attribute__((aligned(ALIGN_SIZE))) = {0, 0, 0, 0};
+    uint16_t _arr[4] __attribute__((aligned(simd::ALIGNMENT))) = {0, 0, 0, 0};
```

Comment:
It is in `simd` namespace so `simd::` is not necessary

Reply:
Good catch! I'll fix this :smiley: